src/serac/physics/base_physics.cpp

```diff
@@ -45,10 +47,14 @@ void BasePhysics::setTime(const double time) { time_ = time; }
 
 double BasePhysics::time() const { return time_; }
 
+double BasePhysics::adjointTime() const { return adjoint_time_; }
+
 void BasePhysics::setCycle(const int cycle) { cycle_ = cycle; }
 
 int BasePhysics::cycle() const { return cycle_; }
 
+int BasePhysics::adjointCycle() const { return adjoint_cycle_; }
```

Comment:
What's the adjoint cycle concept? Is it an iteration count or what?

Reply:
It's the iteration count on the backward pass. I added some extra docs, but let me know if this isn't clear.